src/PowerShell.Core.Instrumentation/PowerShell.Core.Instrumentation.man

```diff
@@ -2264,23 +2265,19 @@
           </channel>
           <channel
               chid="C_ADMIN"
```

Comment:
Since there are no events referencing this channel, it should simply be removed, not disabled.